server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -14,9 +14,21 @@
 import java.util.UUID;
 
 @Entity
-@Table(name ="skills")
+@Table(name = "skills")
 public class Skill {
 
+    @Id
```

Comment:
These are missing schema definition, please add those and add some spacing between variables for better readability.

Reply:
done

Reply:
done